google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -164,28 +163,19 @@ public void checkConfiguration() throws RuntimeConfigurationException {
           "App Engine facet not found in '" + artifact.getName() + "' artifact");
     }
 
-    CloudSdkService cloudSdkService = CloudSdkService.getInstance();
```

Comment:
Why aren't we checking for null here? I think we should because, otherwise, when you build the SDK object, the library is going to try and search for a Cloud SDK on a null path, and that's not what we want.